recipes/oommf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: oommf
+  version: "1.2b0_20160930b1"  # [unix]
+  version: "1.2b0_20160930b0"  # [win]
+
+# on Windows, we install from precompiled binary packages
+source:  # [unix]
+  git_url: https://github.com/fangohr/oommf  # [unix]
+  git_rev: 1.2b0_20160930b1  # [unix]
+
+build:
+  number: 0
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+    - curl  # [win]
+    - 7za  # [win]
+  run:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+
+test:  # [unix]
+  commands:  # [unix]
+    - oommf +version  # [unix]
```

Comment:
I don't know, unfortunately. @fangohr or @mb1a15 might have a better idea, but I don't think any of us are familiar with Windows.